packages/react-searchbox/examples/demo/src/App.js

```diff
@@ -196,6 +210,28 @@ export default () => (
                       ))}
                     </div>
                   )}
+                  <ReactPaginate
+                    pageCount={Math.floor(results.numberOfResults / size)}
+                    onPageChange={({ selected }) =>
+                      setFrom((selected + 1) * size)
+                    }
+                    previousLabel={'previous'}
```

Comment:
@nikhil-varma Why you're using brackets for string props?
Please remove it from everywhere.